lib/swoosh/adapters/sendgrid.ex

```diff
@@ -46,63 +45,58 @@ defmodule Swoosh.Adapters.Sendgrid do
   defp prepare_body(%Email{} = email) do
     %{}
     |> prepare_from(email)
-    |> prepare_to(email)
-    |> prepare_cc(email)
-    |> prepare_bcc(email)
+    |> prepare_personalizations(email)
     |> prepare_subject(email)
-    |> prepare_html_body(email)
-    |> prepare_text_body(email)
+    |> prepare_content(email)
     |> prepare_reply_to(email)
+    |> Poison.encode!
```

Comment:
Can you move this to the `deliver/2` function. Like this:

`body = email |> prepare_body() |> Poison.encode!`

That way we are consistent with the other adapters, and this function is only concerned about building a representation of the body, not how it's encoded.